test/games/strategy/test/MockitoExtension.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Ah, this is a copy/pasted class.. include the copyright only if actually needed. A reference to the original source (URL) would be of help